styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,117 @@
 .footer{
-  
-}
\ No newline at end of file
+  color:#fff;
```

Comment:
Use swatches for colors. example $white. Look at the swatches scss file for those names.